docs/upgrade-guide/upgrade411-50.md

```diff
@@ -7,6 +7,7 @@ Title: Upgrade guides
 Below are links to the upgrade guides notes for all released versions of ADF back to v2.6.0
 
 - [Upgrading from ADF v5.0 to v6.0](upgrade50-60.md)
+- [upgrading from ADF v4.11 to v5.0](upgrade411-50.md)
```

Comment:
I don't see the referenced file anywhere in the files: https://github.com/Alfresco/alfresco-ng2-components/tree/develop/docs/upgrade-guide